generators/app/templates/src/app/components/InputLabel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function InputLabel({
+  className,
+  textClassName,
+  dataFor,
+  label,
+  inputClassName,
+  inputName,
+  placeholder,
+  inputId,
+  inputType,
+  handleChange,
+  disabled
+}) {
+  return (
+    <div className={`column start ${className}`}>
+      <label className={`${textClassName} m-bottom-1`} htmlFor={dataFor}>
+        {label}
+      </label>
+      <input
+        className={inputClassName}
+        name={inputName}
```

Comment:
let's call this prop `name`. So it matches with redux-form's name, and html's `input`'s name attribute